addons/stock/models/stock_move.py

```diff
@@ -1205,7 +1205,7 @@ def _onchange_lot_ids(self):
             for quant in quants:
                 sn_to_location += _("\n(%(serial_number)s) exists in location %(location)s", serial_number=quant.lot_id.display_name, location=quant.location_id.display_name)
             return {
-                'warning': {'title': _('Warning'), 'message': _('Unavailable Serial numbers. Please correct the serial numbers encoded:') + sn_to_location}
+                'warning': {'title': _('Warning'), 'message': _('Unavailable Serial numbers. Please correct the serial numbers encoded: %(serial_numbers_to_locations)s', serial_numbers_to_locations=sn_to_location)}
```

Comment:
the name of this placeholder is way too long. @ticodoo Do you have a better idea?